Sources/MediaCategories/MediaCategoryViewController.swift

```diff
@@ -81,14 +82,30 @@ class VLCMediaCategoryViewController: UICollectionViewController, UICollectionVi
         self.services = services
         self.model = model
         self.rendererButton = services.rendererDiscovererManager.setupRendererButton()
+        self.searchDataSource = LibrarySearchDataSource(model: model)
+
         super.init(collectionViewLayout: UICollectionViewFlowLayout())
+        setupHeader()
         if let collection = model as? CollectionModel {
             title = collection.mediaCollection.title()
         }
         NotificationCenter.default.addObserver(self, selector: #selector(themeDidChange), name: .VLCThemeDidChangeNotification, object: nil)
         navigationItem.rightBarButtonItems = [editButtonItem, UIBarButtonItem(customView: rendererButton)]
     }
 
+    func setupHeader() {
+
```

Comment:
Should this method be renamed to `setupSearchBar`?

Reply:
will do